server/src/auth/auth.controller.ts

```diff
@@ -1,27 +1,28 @@
-import { Controller, Post, Response, Request, UseGuards, Get } from '@nestjs/common';
+import { Controller, Post, Response, Request, Get, BadRequestException } from '@nestjs/common';
 
 import type { Request as RequestT, Response as ResponseT } from 'express';
 
-import { LoginAuthGuard } from './guards/login-auth.guard';
-import { CookieAuthGuard } from './guards/cookie-auth.guard';
 import { AuthService } from './auth.service';
-import { COOKIE_KEY } from './constants';
+import { COOKIE_KEY } from '../acccount-manager/constants';
 
 import type { User } from '../users/entities/user.entity';
 
 type AuthedRequest = RequestT & { user: User };
 
-@Controller('auth')
+/**
+ * @Deprecated @See AccountManagerModule
+ */
+@Controller('deprecated/auth')
 export class AuthController {
   constructor(private authService: AuthService) {}
 
   @Post('login')
-  @UseGuards(LoginAuthGuard)
   async login(
     @Request() request: AuthedRequest,
     @Response({ passthrough: true }) response: ResponseT,
   ): Promise<void> {
     const { user } = request;
+    throw new BadRequestException('Deprecated resource. Use the new account manager module');
```

Comment:
throwing errors in here is the same as removing imo
the impact to the frontend is the same
i vote we either keep these as they were or delete now

Reply:
and same goes for all other endpoints that `throw`

Reply:
Mainly, I left this here to make other engineer's conflict resolution easier.